Source/WebKit/NetworkProcess/NetworkSessionCreationParameters.cpp

```diff
@@ -109,6 +109,9 @@ void NetworkSessionCreationParameters::encode(IPC::Encoder& encoder) const
     encoder << serviceWorkerRegistrationDirectory << serviceWorkerRegistrationDirectoryExtensionHandle << serviceWorkerProcessTerminationDelayEnabled << inspectionForServiceWorkersAllowed;
 #endif
     encoder << resourceLoadStatisticsParameters;
+#if ENABLE(DECLARATIVE_WEB_PUSH)
+    encoder << declarativeWebPushEnabled;
```

Comment:
Missing prefix as per WebKit coding style. Should something like `isDeclarativeWebPushEnabled`.